tests/misc_env/install_prereq.py

```diff
@@ -21,15 +18,15 @@
 log = Log(__name__)
 
 
-class ConfigNotFoundError(Exception):
+class RepoConfigError(Exception):
```

Comment:
Where this exceptions are used ?

Reply:
pls refer the below usage 

            status = sm.subscription_manager_status(ceph)
            if status == "Unknown" or skip_enabling_rhel_rpms:
                log.info("Enabling local RHEL repositories")
                if not setup_local_repos(ceph):
                    raise RepoConfigError("Failed to enable local RHEL repositories")